clients/java/src/main/java/io/camunda/zeebe/client/impl/http/TypedApiEntityConsumer.java

```diff
@@ -136,6 +142,17 @@ public void releaseResources() {
     public int getBufferedBytes() {
       return bufferedBytes;
     }
+
+    private String getJsonString() {
+      try (final ByteArrayOutputStream output = new ByteArrayOutputStream();
+          final JsonGenerator generator = json.createGenerator(output)) {
+        buffer.serialize(generator);
+        generator.flush();
+        return output.toString(StandardCharsets.UTF_8.name());
+      } catch (final Exception ex) {
+        return "Original response cannot be constructed";
```

Comment:
we should pass the exception to the logger, you will thank me when you are the medic 😄 
```suggestion
        LOGGER.error("Failed to serialize the error response", ex);
        return "Original response cannot be constructed";
```

Reply:
Yeah, you're right 😆